### PR TITLE
Minor fixes + Karaite Exilarch

### PR DESCRIPTION
**Pull request by IhateTrains**
- title history fixes
- character history fixes
- religion fixes
- localisation fixes
- added some localisation for missing VIET keys
- code cleaning
- removed duplicate dynasty

### CODE REVIEW

File: WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt [hunk 1]
```diff
@@ -608,6 +608,31 @@ d_exilarch = {
...
+	# mercenary = yes
+
+	# Always exists
+	landless = yes
+	
+	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
+	
+	# mercenary_type = exilarch_karaite_composition
+}
+
```
Comment (loup99) [inline WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: Where did you find this title?

File: WTWSMS/localisation/WtWSMS_nicknames.csv [hunk 2]
```diff
@@ -24,7 +24,7 @@ nick_dicoros;Dicoros;Dicoros;;;;;;;;;;;;x
 nick_coscrach;Coscrach;Coscrach;;;;;;;;;;;;x
 nick_lonn;Lonn;Lonn;;;;;;;;;;;;x
 nick_dub;Dub;Dub;;;;;;;;;;;;x
-nick_the_dismal;the Dismal;;;;;;;;;;;;;x
+nick_simhavikrantagami;Simhavikrantagami;;;;;;;;;;;;;x
```
Comment (loup99) [inline WTWSMS/localisation/WtWSMS_nicknames.csv]: Was it intentional that you removed the other localisation entry?

File: WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt (hunk 1 above)
Comment (IhateTrains) [reply WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: In landed_titles.txt in VIET files

File: WTWSMS/localisation/WtWSMS_nicknames.csv (hunk 2 above)
Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_nicknames.csv]: Yes, nick_the_dismal is already localised

File: WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt (hunk 1 above)
Comment (loup99) [reply WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: The latest public release?

File: WTWSMS/localisation/WtWSMS_nicknames.csv (hunk 2 above)
Comment (loup99) [reply WTWSMS/localisation/WtWSMS_nicknames.csv]: Oh, OK.

File: WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt (hunk 1 above)
Comment (IhateTrains) [reply WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: No, the one I mentioned some time ago

Comment (loup99) [reply WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: I'm not sure on the title, I will have to look into it.

**Comment by IhateTrains** [comment]
Just a minor fix for the AO plot localisation

File: WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt (hunk 1 above)
Comment (IhateTrains) [reply WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: Has it been removed from VIET?

Comment (loup99) [reply WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: I think it was removed in the latest version, but that one didn't include all modules either.

Comment (IhateTrains) [reply WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: I think it should be in:
- karaite religion exists in game and this is their religious head
- this title is referenced in many VIET events

Comment (loup99) [reply WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: I think it should, but if you look at the history you added this title was created in the 8th century.

**Comment by IhateTrains** [comment]
Is the event OK now?

**Comment by loup99** [comment]
Sure! Ideally it would maybe get a bit more interesting MTTH with a few weights, but it is fine for now.

**Comment by IhateTrains** [comment]
There is one thing I want to fix in the event before you merge

**Comment by loup99** [comment]
Okay!

File: WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt [hunk 3]
```diff
@@ -608,6 +608,31 @@ d_exilarch = {
...
+	capital = 693 # Baghdad
+
+	title = "EXILARCH"
+	foa = "KOHEN_GADOL_FOA"
+	short_name = yes
+	
+	# Parent Religion 
+	religion = karaite
+	
+	culture = mizrahi
```
Comment (loup99) [inline WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: Here too, I think you added the reference to the Mizrahi culture by accident.

Comment (IhateTrains) [reply WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: I have changed it to Aramaic

**Comment by IhateTrains** [comment]
The event shold be OK now. I've tested it but it would be best if you also tested it after merging.

File: WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt (hunk 3 above)
Comment (IhateTrains) [reply WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt]: I have changed it to Aramaic

**Comment by loup99** [comment]
Okay, will do!

**Pull request merged**